scripts/tests/lib.sh

```diff
@@ -33,6 +33,23 @@ function client_nslookup() {
     client timeout 30 sh -c "nslookup $1 | tee >(cat 1>&2) | tail -n +4"
 }
 
+function api_send_reject_access() {
+    local site_name="$1"
+    local resource_name="$2"
+
+    docker compose exec -T api bin/api rpc "
+Application.ensure_all_started(:domain)
+account_id = \"c89bcc8c-9392-4dae-a40d-888aef6d28e0\"
+
+[gateway_group] = Domain.Gateways.Group.Query.not_deleted() |> Domain.Gateways.Group.Query.by_account_id(account_id) |> Domain.Gateways.Group.Query.by_name(\"$site_name\") |> Domain.Repo.all()
+[gateway_id | _] = Domain.Gateways.Presence.Group.list(gateway_group.id) |> Map.keys()
+[client_id | _] = Domain.Clients.Presence.Account.list(account_id) |> Map.keys()
+[resource] = Domain.Resources.Resource.Query.not_deleted() |> Domain.Resources.Resource.Query.by_account_id(account_id) |> Domain.Repo.all() |> Enum.filter(&(&1.name == \"$resource_name\"))
+
+Domain.PubSub.Account.broadcast(account_id, {{:reject_access, gateway_id}, client_id, resource.id})
+"
```

Comment:
This is quite a lot of inline Elixir that's not statically checked. How about defining a simple helper function in the Gateway module and just calling that helper function from here?

E.g.



```suggestion
    docker compose exec -T api bin/api rpc "API.Gateway.Channel.send_test_reject_access(\"$resource_name\")"
```

Reply:
> This is quite a lot of inline Elixir that's not statically checked. How about defining a simple helper function in the Gateway module and just calling that helper function from here?

I have no opinion in regards to that. I originally built it like this because it was quicker to iterate on inline Elixir code vs having to recompile constantly.

Reply:
Feel free to push a commit!

Reply:
I do think we want to pass resource and site name unless you know a way how to target the correct gateway.